### PR TITLE
chore: polish notifications menu, markets page, display settings on mobile

### PR DESCRIPTION
**Pull request by aforaleka**
changes:
- added entry point to preferences menu in notifications menu
- remove search input on some combobox menu (preference dialog) and fix styling
- markets page billboards padding + fix height on output with subscript so the market stats align + have same row height
- add entry point to display settings for mobile web
- fix share position dialog icon size

image
image
image
image
image

### CODE REVIEW

File: src/views/menus/NotificationsMenu.tsx [hunk 1]
```diff
@@ -125,7 +133,16 @@ export const NotificationsMenu = ({
       isOpen={isMenuOpen || placement === DialogPlacement.Inline}
       setIsOpen={setIsMenuOpen}
       items={items}
-      title={stringGetter({ key: STRING_KEYS.NOTIFICATIONS })}
+      title={
+        <div tw="flex items-center justify-between">
+          {stringGetter({ key: STRING_KEYS.NOTIFICATIONS })}{' '}
```
Comment (moo-onthelawn) [inline src/views/menus/NotificationsMenu.tsx:138]: nit, remove empty space here

File: src/views/menus/NotificationsMenu.tsx [hunk 2]
```diff
@@ -125,7 +133,16 @@ export const NotificationsMenu = ({
...
+          <IconButton
+            iconName={IconName.Gear}
+            onClick={() => dispatch(openDialog(DialogTypes.Preferences()))}
+            fullIcon
+          />
+        </div>
+      }
```
Comment (moo-onthelawn) [inline src/views/menus/NotificationsMenu.tsx:145]: nit: (existing bug) when I click outside of the preferences dialog on mobile web to dismiss, it auto opens the dialog that pops up from clicking on the notif


https://github.com/user-attachments/assets/e15e4f25-f3d0-42ea-8cfc-6c8b33a64a27

File: src/views/dialogs/SharePNLAnalyticsDialog.tsx [hunk 3]
```diff
@@ -104,7 +104,7 @@ export const SharePNLAnalyticsDialog = ({
       >
         <div tw="flexColumn h-full">
           <div tw="row mb-0.75 gap-0.5">
-            <AssetIcon symbol={assetId} tw="h-[1.625rem]" />
+            <AssetIcon symbol={assetId} tw="h-[1.625rem] w-[1.625rem]" />
```
Comment (moo-onthelawn) [inline src/views/dialogs/SharePNLAnalyticsDialog.tsx:107]: I think we should use `--asset-icon-size` here instead cuz it sets the minWidth and minHeight as well; if we don't wanna do custom css we could add a `size` prop the way `Icon` has it

File: src/views/MarketsDropdown.tsx [hunk 4]
```diff
@@ -173,9 +173,10 @@ const MarketsDropdownContent = ({
           >
             🇺🇸 {stringGetter({ key: STRING_KEYS.TRADE_US_PRESIDENTIAL_ELECTION })} →
           </Link>
-          <$IconButton
+          <IconButton
             onClick={() => setHasSeenElectionBannerTrupmWin(true)}
             iconName={IconName.Close}
+            fullIcon
```
Comment (moo-onthelawn) [inline src/views/MarketsDropdown.tsx:179]: hmm the close button seems sorta big?

Screenshot 2024-10-18 at 11 37 18 AM

File: src/pages/settings/Settings.tsx [hunk 5]
```diff
@@ -101,7 +108,7 @@ const SettingsPage = () => {
           path={MobileSettingsRoute.Network}
           element={<PageMenu group="network" items={[networkMenuItems]} />}
         />
-        <Route path="*" element={<Navigate to="" replace />} />
```
Comment (moo-onthelawn) [inline src/pages/settings/Settings.tsx:104]: oh curious - what did this used to do?

File: src/components/NumberValue.tsx [hunk 6]
```diff
@@ -31,3 +30,8 @@ export const NumberValue = ({ className, value, withSubscript }: NumberValueProp
 
   return value;
 };
+
+const $Sub = styled.sub`
+  font-size: 0.85em;
```
Comment (moo-onthelawn) [inline src/components/NumberValue.tsx:35]: nit: would prefer to add a variable for it in text.css

File: src/components/IconButton.tsx [hunk 7]
```diff
@@ -47,6 +52,7 @@ export const IconButton = forwardRef<HTMLButtonElement | HTMLAnchorElement, Icon
         shape={shape}
         href={href}
         onPressedChange={onPressedChange ?? (onClick as any)} // TODO fix types
+        $fullIcon={fullIcon}
```
Comment (moo-onthelawn) [inline src/components/IconButton.tsx:55]: nit: maybe "withoutBackground" or something instead? idk if that's better

File: src/views/dialogs/PreferencesDialog.tsx [hunk 8]
```diff
@@ -13,6 +13,7 @@ export const PreferencesDialog = ({ setIsOpen }: DialogProps<PreferencesDialogPr
   return (
     <ComboboxDialogMenu
       isOpen
+      withSearch={false}
```
Comment (moo-onthelawn) [inline src/views/dialogs/PreferencesDialog.tsx:16]: unrelated nit but the content and title are not aligned :(((

Screenshot 2024-10-18 at 11 47 45 AM

File: src/pages/settings/Settings.tsx (hunk 5 above)
Comment (aforaleka) [reply src/pages/settings/Settings.tsx:104]: ah added it back!

Comment (aforaleka) [reply src/pages/settings/Settings.tsx:104]: basically catching invalid urls

**Pull request merged**